src/commands/txCommands/util/index.js

```diff
@@ -325,6 +325,35 @@ const resources$ =
 		resourceInfo.sort(lastUpdateComparator).map(resource => resource['slug'])
 	);
 
+const resourceStats$ = Rx.Observable.bindNodeCallback(
+	tx.statisticsMethods.bind(tx)
+);
+
+const resourceCompletion$ = resources$
+	.flatMap(Rx.Observable.from)
```

Comment:
I might not be thinking about it correctly, but I think this line is just passing the input directly to the output - it shouldn't be needed?

Reply:
I'm not 100% certain that I understand the context of this comment after making a few changes, but this line converts a single array value into a stream of values.